typescript-express-reviews/README.md

```diff
@@ -2,8 +2,28 @@
 
 This sample demonstrates integrating Mongoose with [`express`](https://www.npmjs.com/package/express).
 
+## Prerequisites
+
+To run or test this app, have either a stargate running *_OR_* start an AstraDB instance on https://astra.datastax.com/ 
+
+
+## Testing
+Update ASTRA_URI OR the STARGATE_* variables in the .env.test file and then run below
```

Comment:
The `_*` here will likely render badly because of Markdown. I'd also bold the `OR` to be consistent with L7.

Reply:
Fixed the OR similar to L7 and the _* didn't add any styling, so leaving that as is.